app/controllers/series_controller.rb

```diff
@@ -31,7 +31,7 @@ def index
 
     if params[:pseud_id]
       @pseud = @user.pseuds.find_by!(name: params[:pseud_id])
-      @page_subtitle = ts("by ") + @pseud.byline
+      @page_subtitle = t(".page_title", username: params[:pseud_id])
```

Comment:
It would be a bit cleaner to use `@pseud.name` instead.

Reply:
done